src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewModel.swift

```diff
@@ -221,36 +221,64 @@ extension ExposureSubmissionTestResultViewModel {
-								description: AppStrings.ExposureSubmissionResult.PCR.testPendingDesc,
-								icon: UIImage(named: "Icons_Grey_Wait"),
-								hairline: .none
-							)
+				ExposureSubmissionDynamicCell.stepCell(
+					title: AppStrings.ExposureSubmissionResult.PCR.testAdded,
+					description: nil,
+					icon: UIImage(named: "Icons_Grey_Check"),
+					hairline: .iconAttached
+				),
+				ExposureSubmissionDynamicCell.stepCell(
+					title: AppStrings.ExposureSubmissionResult.PCR.testPending,
+					description: AppStrings.ExposureSubmissionResult.PCR.testPendingDesc,
+					icon: UIImage(named: "Icons_Grey_Wait"),
+					hairline: .none
+				)
 			])
 		case .antigen:
-			cells.append(contentsOf: [
-							ExposureSubmissionDynamicCell.stepCell(
-								title: AppStrings.ExposureSubmissionResult.Antigen.testAdded,
-								description: nil,
-								icon: UIImage(named: "Icons_Grey_Check"),
-								hairline: .iconAttached
-							),
-							
-							ExposureSubmissionDynamicCell.stepCell(
-								title: AppStrings.ExposureSubmissionResult.Antigen.testPending,
-								description: AppStrings.ExposureSubmissionResult.Antigen.testPendingDesc,
-								icon: UIImage(named: "Icons_Grey_Wait"),
-								hairline: .none
-							)
-			])
+				cells.append(contentsOf: [
+					ExposureSubmissionDynamicCell.stepCell(
+						title: AppStrings.ExposureSubmissionResult.Antigen.testAdded,
+						description: nil,
+						icon: UIImage(named: "Icons_Grey_Check"),
+						hairline: .iconAttached
+					),
+					ExposureSubmissionDynamicCell.stepCell(
+						title: AppStrings.ExposureSubmissionResult.Antigen.testPending,
+						description: AppStrings.ExposureSubmissionResult.Antigen.testPendingDesc,
+						icon: UIImage(named: "Icons_Grey_Wait"),
+						hairline: .iconAttached
+					)
+				])
+			if let test = coronaTest.antigenTest {
+				if !test.certificateSupportedByPointOfCare {
+					cells.append(
+						ExposureSubmissionDynamicCell.stepCell(
+							title: AppStrings.ExposureSubmissionResult.testCertificateTitle,
+							description: AppStrings.ExposureSubmissionResult.Antigen.testCenterNotSupportedTitle,
+							icon: UIImage(named: "certificate-qr-light"),
+							hairline: .none
+						)
+					)
+				} else if test.certificateCreated {
+					cells.append(
+						ExposureSubmissionDynamicCell.stepCell(
+							title: AppStrings.ExposureSubmissionResult.testCertificateTitle,
+							description: AppStrings.ExposureSubmissionResult.Antigen.testCertificatePending,
+							icon: UIImage(named: "certificate-qr-light"),
+							hairline: .none
+						)
+					)
+				} else {
+					cells.append(
+						ExposureSubmissionDynamicCell.stepCell(
+							title: AppStrings.ExposureSubmissionResult.testCertificateTitle,
+							description: AppStrings.ExposureSubmissionResult.Antigen.testCertificateNotRequested,
+							icon: UIImage(named: "certificate-qr-light"),
+							hairline: .none
+						)
+					)
+				}
+			}
```

Comment:
```suggestion
				} else if !test.certificateConsentGiven {
					cells.append(
						ExposureSubmissionDynamicCell.stepCell(
							title: AppStrings.ExposureSubmissionResult.testCertificateTitle,
							description: AppStrings.ExposureSubmissionResult.Antigen.testCertificateNotRequested,
							icon: UIImage(named: "certificate-qr-light"),
							hairline: .none
						)
					)
				} else if !test.certificateRequested {
					cells.append(
						ExposureSubmissionDynamicCell.stepCell(
							title: AppStrings.ExposureSubmissionResult.testCertificateTitle,
							description: AppStrings.ExposureSubmissionResult.Antigen.testCertificatePending,
							icon: UIImage(named: "certificate-qr-light"),
							hairline: .none
						)
					)
				}
			}
```

Reply:
I cant dind the property `test.certificateRequested`  in it inside the **AntigenTest** struct ?

<img width="569" alt="Screenshot 2021-06-07 at 09 44 38" src="https://user-images.githubusercontent.com/15270737/120978663-02fa6f00-c775-11eb-8121-6f4fc575a792.png">

Reply:
Sorry, on your branch it's called `certificateCreated`, I just refactored it on mine to make the naming more accurate. You can just use `certificateCreated` here